scripts/zones/Lower_Jeuno/npcs/Ghebi_Damomohe.lua

```diff
@@ -53,22 +26,6 @@ entity.onEventFinish = function(player, csid, option, npc)
         }
```

Comment:
This is not related to your PR, but, while you are touching this file, could you please add a TODO comment for this items to be tabled to use xi.item enums?

No need to table them yourself unless you REALLY want to.